es/components/browse/components/table-commons/HeadersRow.js

```diff
@@ -149,6 +149,25 @@ export var HeadersRow = /*#__PURE__*/function (_React$PureComponent) {
      * linkTo fields are appended by .display_title by backend so we trim it to find a match
      */
 
+  }, {
+    key: "getSortDirectionBySchemaFieldType",
+    value: function getSortDirectionBySchemaFieldType(fieldType) {
+      switch (fieldType) {
+        case 'string':
+          return 'asc';
+
+        case 'integer':
+          return 'desc';
```

Comment:
In my opinion the expected default order for integer and number should be ascending, whereas special cases like "% accuracy" could be overwritten as desc. But I am not familiar with CGAP schemas, maybe there the majority of numeric properties is the other way around?

Reply:
@andreacosolo we may want to do a quick and informal review to see what makes most sense as default.  I'm thinking that often you do want number fields such as score or percent to initially show up desc and int fields could go either way - eg. replicate number or channel would want asc and number of experiments (if you can even sort on that field) desc - so a quick scan of columns in schema might provide guidance?  It may prove that CGAP and 4DN do want different default behavior.  As a first pass to get this change in I would propose leaving number DESC and change int to ASC but there may be important use cases so that the settings @utku-ozturk chose are sensible and schema changes can be made for fourfront.